### ISSUE TITLE
Update dependencies for spring-web and spring-hateoas

### ISSUE THREAD

**Issue by abeag** [ununbium/HateoasUtils#16]
Please update dependencies up to versions:
```
<spring-web.version>5.2.8.RELEASE</spring-web.version>
<spring-hateoas.version>1.1.0.RELEASE</spring-hateoas.version>
```
 or higher. And publish HateoasUtils 3.x to maven repo.

I am using` 'org.springframework.boot:spring-boot-starter-hateoas:2.3.0.RELEASE'` and bump the issue similar to
(https://github.com/ununbium/HateoasUtils/issues/14)
Downgrade to older Spring Boot is not the case due to I need Hateoas 1.1.0.RELEASE

Thank you

**Comment by ununbium** [comment]
Updated to the following (latest release of requested);
```
        <spring-web.version>5.2.9.RELEASE</spring-web.version>
        <spring-hateoas.version>1.1.2.RELEASE</spring-hateoas.version>
```

Released as **3.1.0** (I hit some issues with the 3 release).

All the best,
Andy

**Comment by abeag** [comment]
Thank you Andy for great utils.
Now it works for me.
